lib/Doctrine/ORM/Configuration.php

```diff
@@ -64,6 +79,8 @@ public function getProxyDir()
      * Gets the strategy for automatically generating proxy classes.
      *
      * @return int Possible values are constants of Doctrine\ORM\Proxy\Factory\StaticProxyFactory.
+     *
+     * @deprecated please do not use this anymore
```

Comment:
Should we drop this?

Reply:
Not removed: too commonly used, therefore keeping it and documented as deprecated